src/components/admin/MainIssueController.jsx

```diff
@@ -156,8 +156,26 @@ export default class MainIssueController extends FalcorController {
               return false;
             }
           }
+          if (/<a.*?href\s*?=\s*?["'](?!http).*?>/.test(article.html)) {
+            if (!window.confirm(
+              `${article.title} is in non-absolute format,
+              which means that it does not have http(s):// in front of it. ` +
+              'This problem will cause the link to break in the' +
```

Comment:
It's a nice description, but it could possibly be shortened a bit by removing these 2 sentences about why it breaks and then you can maybe just append to the first sentence as follows: `...http(s):// in front of it, which will break the link.`